dlatk/dlaConstants.py

```diff
@@ -274,6 +274,16 @@ def fit_transform(self, X, y=None):
     lambda d: 2*sqrt(d+3/float(8))
 ]
 
+## PyMallet defaults
+DEF_STOPLIST_LANGUAGE_CHOICES = ['en', 'de', 'fi', 'fr', 'jp', 'nl']
+DEF_STOPLIST_LANGUAGE = 'en'
```

Comment:
We also have our regular expression for stoplists that would be good to get in here. 

Further, given the wide ranging domains we use, it might be best to just default to removing the top ~50 most common words the stoplist rather than using this. In reality, it's the very head of the Zipfian distribution that causes problems for LDA. If the stopword isn't mentioned a lot it won't cause issues. It just so happens old stopword lists happen to mostly cover that for traditional domains so most people don't realize this. We could set this "head of distribution" number automatically as some function of the entire distribution.